### PR TITLE
Updates for new version of farm card

### PR DESCRIPTION
**Pull request by RichardPK**
- Added `Minus` svg icon for new farm card 'withdraw' button
- Added `textTransform` prop to text to allow for it to receive a textTransform argument (i.e. 'uppercase' for the new card text)

### CODE REVIEW

**Comment by RichardPK** [comment]
I don't think the snapshot fails here are anything to do with my changes?

**Comment by RabbitDoge** [comment]
@RichardPK please fix the tests before we review

File: src/components/Text/types.ts [hunk 1]
```diff
@@ -4,4 +4,5 @@ export interface TextProps extends SpaceProps {
   color?: string;
   fontSize?: string;
   bold?: boolean;
+  textTransform?: string;
```
Comment (RabbitDoge) [inline src/components/Text/types.ts:7]: you can narrow down the values to `"uppercase" | "lowercase" | "capitalize"`, or maybe use a boolean `uppercase` ?

**Review by RabbitDoge — CHANGES_REQUESTED** [review]
don't forget to run `yarn test` and check the snapshot diff. then run `yarn test -u` to update it it seems correct to you.

File: src/components/Text/types.ts (hunk 1 above)
Comment (RichardPK) [reply src/components/Text/types.ts]: Aye great shout, sorry I should have had that in from the start. Pro the strings > an `uppercase` boolean

**Comment by RichardPK** [comment]
Updated the snaps @RabbitDoge - it was class hashes, which shouldn't really be there in the first place. I think you'll have _a lot_ of redundant snapshot changes unless this is fixed. You may need to look at the repo's `jest-styled-components` setup to resolve this?

bad:
```
 <button
     className="sc-bdVaJa rOCEJ"
 />
```

good:
```
 <button
     className="c0"
 />
```

**Review by RabbitDoge — APPROVED** [review]

**Pull request merged**